spec/index.bs

```diff
@@ -6238,30 +6252,33 @@ Issue: convert "Valid Texture Copy Range" into an algorithm with parameters, sim
 
             **Returns:** {{undefined}}
 
-            If any of the following conditions are unsatisfied, generate a validation error and stop.
-            <div class=validusage>
-                - Let |srcTextureDesc| be |source|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[descriptor]]}}.
-                - |this|.{{GPUCommandEncoder/[[state]]}} is [=encoder state/open=].
-                - [$validating GPUImageCopyTexture$](|source|, |copySize|) returns `true`.
-                - |srcTextureDesc|.{{GPUTextureDescriptor/usage}} contains {{GPUTextureUsage/COPY_SRC}}.
-                - |srcTextureDesc|.{{GPUTextureDescriptor/sampleCount}} is 1.
-                - If |srcTextureDesc|.{{GPUTextureDescriptor/format}} is a depth-stencil format:
-                    - |destination|.{{GPUImageCopyTexture/aspect}} must refer to a single aspect of
-                        |srcTextureDesc|.{{GPUTextureDescriptor/format}}, and that aspect must be
-                        a valid image copy source according to [[#depth-formats]].
-                - [$validating GPUImageCopyBuffer$](|destination|) returns `true`.
-                - |destination|.{{GPUImageCopyBuffer/buffer}}.{{GPUBuffer/[[usage]]}} contains
-                    {{GPUBufferUsage/COPY_DST}}.
-                - [=Valid Texture Copy Range=] applies to |source| and |copySize|.
-                - If |srcTextureDesc|.{{GPUTextureDescriptor/format}} is not a [=depth-or-stencil format=]:
-                    - |destination|.{{GPUImageDataLayout/offset}} is a multiple of the [=texel block size=] of
-                        |srcTextureDesc|.{{GPUTextureDescriptor/format}}.
-                - If |srcTextureDesc|.{{GPUTextureDescriptor/format}} is a [=depth-or-stencil format=]:
-                    - |destination|.{{GPUImageDataLayout/offset}} is a multiple of 4.
-                - [$validating linear texture data$](|destination|,
-                    |destination|.{{GPUImageCopyBuffer/buffer}}.{{GPUBuffer/[[size]]}},
-                    |srcTextureDesc|.{{GPUTextureDescriptor/format}},
-                    |copySize|) succeeds.
+            Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
+            <div class=device-timeline>
+                1. [$Validate the encoder state$] of |this|. If it returns false, stop.
+                1. If any of the following conditions are unsatisfied, generate a validation error and stop.
```

Comment:
a bit unfortunate that all of the heavy lists are now indented more
but it doesn't look like there is an easy way to solve this

Reply:
I may have some thoughts on improving this in the future, but for now IMO it renders fine.